src/core/core.js

```diff
@@ -32,6 +32,11 @@ module.exports = function() {
 			// Element defaults defined in element extensions
 			elements: {},
 
+			// Layout options such as padding
+			layout: {
+				padding: 0
```

Comment:
I think it's better to expand the default option to the object form to get a better options merge?

```javascript
// defaults:
layout: {
    padding: {
        top: 0,
        right: 0,
        bottom: 0,
        left: 0
    }
}

// merged with:
layout: {
    padding: {
        top: 16
    }
}

// produces:
layout: {
    padding: {
        top: 16,
        right: 0,
        bottom: 0,
        left: 0
    }
}

// instead of:
layout: {
    padding: {
        top: 16
        //right: undefined
        //bottom: undefined
        //left: undefined
    }
}
```